lib/middleware/karma.js

```diff
@@ -193,7 +193,11 @@ function createKarmaMiddleware (
             } else {
               const scriptType = (SCRIPT_TYPE[fileType] || 'text/javascript')
               const crossOriginAttribute = includeCrossOriginAttribute ? 'crossorigin="anonymous"' : ''
-              scriptTags.push(`<script type="${scriptType}" src="${filePath}" ${crossOriginAttribute}></script>`)
+              if (fileType === 'module') {
```

Comment:
Why the `if`.  Is there a reason to to have the `onerror` for both paths?

Reply:
I did that that way to reduce the impact... I wonder if the error would show up twice without the if

Reply:
I did make the test. Without the if, there are tests that does not pass anymore (5 in the first phase). So the "if" is necessary to reduce the impact of the change.